kubeadm/1.29/kcl.mod

```diff
@@ -0,0 +1,5 @@
+[package]
+name = "kubeadm"
+edition = "*"
+version = "1.29+kcl0"
```

Comment:
Is this a valid version number? if not ,I'm looking to iterate multiple times on kubeadm 1.29, any suggestions?

Reply:
cc @zong-zhe

Reply:
Hi @liangyuanpeng 😊

Is the `version` field value `1.29+kcl0` generated by `kcl` command ? We currently require that the package version number of KCL support the semantic version, if this version is generated, this should be a bug, welcome to open an issue and provide some more details.

the semantic version - https://semver.org/
raise an issue - https://github.com/kcl-lang/kpm/issues/new?assignees=&labels=type%2Fbug&projects=&template=bug-report.md

Reply:
@zong-zhe 

> Is the version field value 1.29+kcl0 generated by kcl command

No, This is the version format I added myself

> the semantic version - https://semver.org/

Thanks, I have check it and it's not match, and `1.29.0+kcl0` is match, let me update it.